modules/test/playwright/playwright.config.ts

```diff
@@ -5,13 +5,14 @@
 
 import {defineConfig} from '@playwright/test';
 
+import {config as FDS} from './tests/FDS/config';
 import {config as setup} from './tests/global.setup.config';
 import {config as object} from './tests/object/config';
 import {config as portalWeb} from './tests/portal-web/config';
 
 export default defineConfig({
 	forbidOnly: !!process.env.CI,
-	projects: [object, portalWeb, setup],
+	projects: [FDS, object, portalWeb, setup],
```

Comment:
Should be `fds`, `frontendDataSet`, or anything along those lines

Reply:
dsm or dSM, then :sweat_smile: 

In any case, we have a frontend guideline which says: don't use abbreviations because they are hard to understand, so we should expand it to something larger.

Reply:
For me, `dataSetManager` is the best candidate so far

Reply:
`dataSetManager ` is the best candidate for me too.

Reply:
I'm also adding `Config` suffix in my PR, to be more specific. With it, it can be something like:
```js
import {config as dataSetManagerConfig} from './tests/frontend-data-set-views/config';
```